packages/x-components/src/views/home/Home.vue

```diff
@@ -378,21 +378,32 @@
                         </template>
 
                         <template #next-queries-group="{ item: { nextQueries } }">
-                          <div class="x-list x-list--gap-03">
-                            <h1 class="x-title2">What's next?</h1>
-                            <BaseSuggestions
-                              #default="{ suggestion }"
+                          <div class="x-list x-padding--06 x-background--neutral-95">
+                            <div class="x-padding--00">
+                              <h1 class="x-title2 x-text--bold x-padding--bottom-03">
```

Comment:
change the `padding-bottom` by `gap` in parent, this way it has better maintainability. If we add more elements, they will use the same separation without adding more CSS.